[2023-03-30 09:57:33,366] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:33] "[31m[1mPOST /dog HTTP/1.1[0m" 422 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:33,379] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:33] "[31m[1mPOST /dog HTTP/1.1[0m" 422 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:33,565] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:33] "[31m[1mPOST /dog HTTP/1.1[0m" 422 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:33,752] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:33] "[31m[1mPOST /dog HTTP/1.1[0m" 422 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:54,477] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:54] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:58:01,622] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:58:01,623] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:58:01] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:58:01,642] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:58:01] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:58:09,175] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:58:09] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:58:15,525] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:58:15,526] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:58:15] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:58:15,533] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:58:15] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:58:39,913] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:58:39] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:58:53,204] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:58:53] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:59:05,570] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:59:05,570] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:59:05] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:59:25,883] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:59:25] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:59:30,229] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:59:30,229] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:59:30] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:00:01,829] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:00:01] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:00:09,775] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 3', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 10:00:09,776] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:00:09] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:00:11,418] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:00:11] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:00:17,733] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:00:17] "POST /dog HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:00:19,218] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:00:19] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:00:20,245] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:00:20] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:01:40,323] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:01:40] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:01:44,839] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:01:44] "POST /dog HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:01:46,450] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:01:46] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:01:51,905] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:01:51] "GET /cats HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:01:57,943] WARNING add_cat() L135  Erro ao adicionar gato 'Gato 1', Gato de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L135 
[2023-03-30 10:01:57,944] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:01:57] "[31m[1mPOST /cat HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:01:59,497] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:01:59] "GET /cats HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:02:06,387] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:02:06] "POST /cat HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:02:07,730] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:02:07] "GET /cats HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:02:12,492] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:02:12] "OPTIONS /cat?id=3 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:02:12,509] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:02:12] "DELETE /cat?id=3 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:04:27,142] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:04:27] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:04:42,514] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:04:42] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:04:42,516] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 10:04:42,517] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:04:42] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:05:14,029] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:05:14] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 10:05:18,774] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 10:05:18,775] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 10:05:18] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
